package.json

```diff
@@ -32,6 +32,8 @@
   },
   "files": [
     "lib",
+    "docs",
+    "lib/**/*",
```

Comment:
```suggestion
    "lib",
```

Reply:
Fixed in https://github.com/ridedott/eslint-plugin/pull/30/commits/0d1b41b2e6fa7c18ddca54619c872d55e5350a02